### ISSUE TITLE
New F# project in VS for Mac shows intellisense errors

### ISSUE THREAD

**Issue by willsam100** [mono/monodevelop#6757]
## Steps to Reproduce

1. create an F# solution eg Xamarin.Forms app in VS4Mac
2. add a new project eg .NETStandard2.0 project
3. see that the F# file in the new project shows red lines by the editor

### Current Behavior

See that the F# file in the new project shows red lines by the editor. Errors are shown parsing the F# syntax. 
Note the project does compile. 

<img width="476" alt="screen shot 2018-12-11 at 5 37 45 am" src="https://user-images.githubusercontent.com/732506/49752270-bc973700-fd14-11e8-8e10-d1fb32c36322.png">

From the screenshot attached, it would appear that when a new project is created, the syntax parser is using C# as it is expected a `{` after the namespace

### Expected Behavior

No syntax highlighting errors when adding a new project to a solution in VS4Mac

## On which platforms did you notice this

[x ] macOS
[ ] Linux
[ ] Windows

**Version Used**:
```
Visual Studio Community 2017 for Mac
Version 7.7.1 (build 15)
Installation UUID: 8d3712bd-ee56-4387-80b1-d5ddc495dce3
	GTK+ 2.24.23 (Raleigh theme)
	Xamarin.Mac 4.4.1.178 (master / eeaeb7e6)

	Package version: 516000221

Mono Framework MDK
Runtime:
	Mono 5.16.0.221 (2018-06/b63e5378e38) (64-bit)
	Package version: 516000221
```
## Known work-around

unload the entire solution, and reloading the solution fixes this issue.

> VS bug [#748732](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/748732)

**Comment by nosami** [comment]
@willsam100 I missed the cut for 7.7.2, but this should be fixed for 7.7.3

**Comment by nosami** [comment]
@monojenkins backport release-7.7

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-12] Comment by **xamarinc**



Fixed in version 7.8.0.1416 (master)



Pull Request #6764 merged by: Lluis Sanchez  
 Author: mono  
 Commit: 120bc8a297016e8e035db5374ed161d9b534dd39 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-13] Comment by **xamarinc**



Fixed in version 7.8.0.1460 (release-7.8)



Pull Request #6764 merged by: Lluis Sanchez  
 Author: mono  
 Commit: 120bc8a297016e8e035db5374ed161d9b534dd39 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-15] Comment by **xamarinc**



Fixed in version 8.0.0.973 (release-8.0)



Pull Request #6764 merged by: Lluis Sanchez  
 Author: mono  
 Commit: 120bc8a297016e8e035db5374ed161d9b534dd39 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-17] Comment by **xamarinc**



Fixed in version 7.7.3.22 (release-7.7)



Pull Request #6766 merged by: Lluis Sanchez  
 Author: mono  
 Commit: f507aba1f8ac17540968ef4c1fde28d4724fb90f (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-20] Comment by **xamarinc**



Fixed in version 8.0.0.1738 (release-8.0)



Author: nosami  
 Commit: 7fdf47558fe27ad0d040eb819bb75cb7160602bc (xamarin/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-26] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #6764 merged by: Lluis Sanchez  
 Author: mono  
 Commit: 120bc8a297016e8e035db5374ed161d9b534dd39 (xamarin/monodevelop)
